main.go

```diff
@@ -244,19 +243,26 @@ func createMilestones(baseURL string, token string, projectID string, milestones
 		if err != nil {
 			return err
 		}
-
 		req.Header.Add("PRIVATE-TOKEN", token)
 		resp, err := client.Do(req)
 		if err != nil {
 			return err
 		}
-
 		defer resp.Body.Close()
 	}
 
 	return nil
 }
 
+func createMilestoneMap(milestones []milestoneAPI) (map[string]milestone) {
+	var m milestone
+	activeMilestones := map[string]milestone{}
+	m.DueDate = "2018-01-01"
```

Comment:
This hard codes the DueDate. You want to get the list of milestoneAPI objects and iterate through all of them and then set the DueDate, ID and Title to their respective value.